organizational/conduct.rst

```diff
@@ -0,0 +1,101 @@
+The Mission of Hacker Society
+=============================
+
+A time and place for people to learn about technology from each other, in order
+to build a community, to empower the individual, promote equality, and build an
+open society.
+
+**Open Society**: An open society promotes freedom of thought and expression,
+personal responsibility, individualism, cooperation, and equality of all
+citizens.
+
+Code of Conduct
+===============
+
+It is expected that members behave in a way that allows Hacker Society to
+achieve its mission.  Participants in Hacker Society events are expected to
+treat each other with dignity and respect for each other's perspectives,
+experiences, and knowledge.  In particular, participants should recognize that
+people of all skill levels are welcome, and should always be willing to help
+others learn.
+
+To support Hacker Society's mission of: learning technology from each other,
+building community, and promoting equality, Hacker Society must be a place where
+people can be comfortable regardless of gender, gender identity and expression,
+sexual orientation, disability, physical appearance, body size, race, age, or
+religion. Therefore, we do not tolerate:
+
+- Rude or unwelcome comments about a person's gender, gender identity, gender
+  expression, sexual orientation, disability, physical appearance, attire, race,
+  age or religious expression.
+- Sexual images in public spaces
+- Deliberate intimidation, stalking, or following
+- Harassing photography or recording
+- Sustained disruption of talks or other events
+- Inappropriate physical contact
+- Unwelcome sexual attention
+- Advocating for, or encouraging, any of the above behavior
+
+Necessarily, the goals of learning, community, and equality are in tension
```

Comment:
> For instance, a person who is blatantly sexist, and believes women can't code, has an extremely unpopular opinion, (one which I find very difficult to accept or justify). I suppose they could be allowed to participate, but the moment they voice that opinion, they are attacking the intrinsic characteristics of all women in our group, and that's not okay to me, nor is it acceptable by this code of conduct.

Agreed. That situation is one this CoC is meant to specifically address (and I think it does). If they do voice that opinion they would need to leave and not be invited back.

> I would tend to agree with this. In reality it is also worth considering the possible PR implications of hosting a talk by somebody with such views. But that should not necessarily be covered by this CoC.

Most of the time we wouldn't know beforehand. In the case that we did know about it before hand the maintainer would need to address the issue head on during the setup of the talk and also consider whether given the other proposals the topic is worth hearing. But in the case where some one is already scheduled for a talk and a potential issue comes to light what is the right course of action? The talk should still occur but the maintainer should before hand ask for a second confirmation of the CoC highlighting their concerns.